Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -266,7 +276,7 @@ public override void HandleEvent<T>(BaseEventData eventData, ExecuteEvents.Event
             // Get the focused object for each pointer of the event source
             for (int i = 0; i < baseInputEventData.InputSource.Pointers.Length; i++)
             {
-                GameObject focusedObject = FocusProvider?.GetFocusedObject(baseInputEventData.InputSource.Pointers[i]);
+                GameObject focusedObject = getFocusedObject(baseInputEventData.InputSource.Pointers[i]);
```

Comment:
`getFocusedObject` is this a method?

Reply:
Yes. I head to split OnFocus events off of the regular HandleEvent call, because for the exit FocusProvider?.GetFocusedObject will return null. For this case I passed the focused object along with it through a lamda to this.

Reply:
Minor issue: MRTK code style capitalizes the first letter of method names.

Reply:
it's a System.Action parameter. does it supply here too?